src/Display/Modal/Modal.test.tsx

```diff
@@ -75,26 +77,32 @@ describe('when modal is opened', () => {
   it('should display the footer', () => {
     const { getByText } = render(OpenedModal);
     const modalFooter = getByText(props.footer);
-    expect(modalFooter).toHaveTextContent(props.footer);
+    expect(modalFooter).toContainHTML(`<button>${props.footer}</button>`);
```

Comment:
is this equivalent to _testing for implementation detail_?

Reply:
Yeah it is. I don't think there is any other way to test for the `button` though. I think I prefer to leave it as `toHaveTextContent(props.footer);` in this case.